pkg/webhook/cmd/options.go

```diff
@@ -146,9 +199,19 @@ func (c *AddToManagerConfig) AddToManager(mgr manager.Manager) error {
 	}
 
 	webhookServer := mgr.GetWebhookServer()
+	webhookServer.CertDir = c.Server.CertDir
+
+	for _, wh := range webhooks {
+		webhookServer.Register("/"+wh.Name, wh.Webhook)
```

Comment:
Maybe use `filepath.Join`

Reply:
This is not a file path, it's a URL. Good enough imho.